### PR TITLE
internet-access-with-iwd

### PR DESCRIPTION
**Pull request by SorrowEater**
So after nearly a month of trying, I added iwd to snakeware! (It wasn't that hard, I just didn't try very hard) With popen, you can integrate this into an app!

### CODE REVIEW

**Comment by Abdur-rahmaanJ** [comment]
Seems impresive

**Comment by joshiemoore** [comment]
I see that you deleted the buildroot config and replaced it with a new one, this makes it difficult to see what was changed. The two files look very different, and a lot of configuration specific to our snakeware build appears to have been removed. Is there any way you could open another PR, changing only what is necessary to add iwd?

File: snakeware/external/configs/x86-64_defconfig [hunk 1]
```diff
@@ -39,45 +494,3211 @@ BR2_PACKAGE_ALSA_UTILS_ASEQNET=y
...
+# BR2_PACKAGE_IMAGEMAGICK is not set
+# BR2_PACKAGE_LINUX_FUSION is not set
+# BR2_PACKAGE_MESA3D is not set
+# BR2_PACKAGE_OCRAD is not set
+
+#
+# ogre needs X11 and an OpenGL provider
+#
+# BR2_PACKAGE_PSPLASH is not set
+# BR2_PACKAGE_SDL is not set
```
Comment (donno2048) [inline snakeware/external/configs/x86-64_defconfig:837]: Isn't it a problem? I thought SDL is required for buildroot's audio functionality and there is no `BR2_PACKAGE_SDL_SOUND=1`

File: snakeware/external/configs/x86-64_defconfig [hunk 2]
```diff
@@ -39,45 +494,3211 @@ BR2_PACKAGE_ALSA_UTILS_ASEQNET=y
...
+#
+# granite needs libgtk3 and a toolchain w/ wchar, threads
+#
 BR2_PACKAGE_GRAPHITE2=y
+
+#
+# gtkmm3 needs libgtk3 and a toolchain w/ C++, wchar, threads, gcc >= 4.9
+#
 BR2_PACKAGE_HARFBUZZ=y
-BR2_PACKAGE_LIBJPEG=y
```
Comment (donno2048) [inline snakeware/external/configs/x86-64_defconfig:70]: Why did you remove this line?

Comment (SorrowEater) [reply snakeware/external/configs/x86-64_defconfig:70]: Honestly, I didn't mean to... Some of the initial config editing was through the official buildroot config, and that must have messed a few things up... I only really need to change one, maybe two lines.

Comment (donno2048) [reply snakeware/external/configs/x86-64_defconfig:70]: There are no jpeg files in Snakeware so this probably non-essential...

**Comment by SorrowEater** [comment]
@joshiemoore Well, I can directly edit the file in GitHub, but there are so many lines needed for packages (if you select one, the not set comment appears for every single package that wasn't chosen), that it's much easier to do it like this. For now, I'll rename the old one "x86-64_defconfig.old", and if this pull request is merged, we can delete it later if needed.
